.travis.yml

```diff
@@ -3,8 +3,7 @@ os:
     - linux
     - osx
 julia:
-    - 0.4
-    - 0.5
+    - 0.7
     - nightly
```

Comment:
the only real difference with #11 is that i allowed failures on nightly to make hexagons like gadfly and compose.